tests/tests_suite_test.go

```diff
@@ -119,7 +119,7 @@ var _ = ReportAfterSuite("TestTests", func(report Report) {
 	}
 })
 
-var _ = ReportBeforeEach(func(specReport SpecReport) {
+var _ = JustBeforeEach(func() {
 	k8sReporter.JustBeforeEach(CurrentSpecReport())
```

Comment:
it will print it every test now isnt it ?
```
func (r *KubernetesReporter) JustBeforeEach(specReport types.SpecReport) {
	fmt.Fprintf(GinkgoWriter, "On failure, artifacts will be collected in %s/%d_*\n", r.artifactsDir, r.failureCount+1)
}

```

Reply:
Like before...

Reply:
yes thats right, it just annoying in case we run on verbose (which was also the state before IIRC)
Maybe worth to print it only if the test fails, but as long as there is no regression its ok

Reply:
I will do a follow up PR printing only on failure.